pkg/shell/shell.html

```diff
@@ -1529,6 +1544,58 @@ <h4 class="modal-title" id="account-confirm-delete-title"></h4>
       </div>
     </div>
 
+    <script id="authorized-keys-tmpl" type="x-template/mustache">
+      {{#keys}}
+        <tr>
+        <td>{{comment}}</td>
```

Comment:
This column doesn't seem to be used.
Is it planned? Otherwise it seems a bit wasteful.